client/webserver/site/src/html/markets.tmpl

```diff
@@ -172,6 +172,9 @@
                   </div>
                 </div>
 
+                {{- /* ORDER PREVIEW INFO */ -}}
+                <div class="mt-2 fmt-2 fs15 text-right" id="orderPreview"></div>
```

Comment:
A couple of things here. 

1. Please use a fixed height div so the layout doesn't change when the value is populated.
2. `fmt-2` appears to be nothing.
3. I think it looks better with `fs14`.